_pages/dev/components/shared-components/media-component.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Media Component
+feature:
+- name: Media Component
+  spa_version: 2.0
+  cx_version: n/a
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 2.0 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The media component is a low-level component that is used to render a single media item. Although many types of media exist, such as images, videos, PDFs, and so on, the media component currently only renders images. The media component renders specific images for different screen sizes and resolutions, so that each user has an optimized version of the image.
```

Comment:
> Although many types of media exist, such as images, videos, PDFs, and so on, the media component currently only renders images.

Can we change to something like: 
Although the backend could provide any type of media for a media item, the media component is currently limited to render images only. The type of image is _not_ limited to a technical format.

Reply:
👍